srcpkgs/j4-dmenu-desktop/template

```diff
@@ -4,7 +4,18 @@ version=2.18
 revision=1
 wrksrc="${pkgname}-r${version}"
 build_style=cmake
-configure_args="-DWITH_TESTS=OFF"
+# The current version (2.18) needs to have /usr/share/applications dir
+# for tests, xterm creates and populates it with its .desktop files,
+# which fixes tests in case the dir does not exist.
+# https://github.com/enkore/j4-dmenu-desktop/pull/123
+checkdepends="catch2 xterm"
+
+if [ "$XBPS_CHECK_PKGS" ]; then
+	configure_args="-DWITH_GIT_CATCH=OFF -DCATCH_INCLUDE_DIR=/usr/include/catch2"
+else
+	configure_args="-DWITH_TESTS=OFF"
+fi
```

Comment:
This entire block should go after the `checksum` line, too.

And `checkdepends` goes below `depends`

Reply:
Ok, thanks!